package.json

```diff
@@ -22,7 +22,7 @@
     "lodash.assign": "^4.0.8"
   },
   "peerDependencies": {
-    "react": "^0.14.8"
+    "react": "^0.14.8 | ~15.0.0"
```

Comment:
Mhm, that'd be kinda intuitive.

I guess I was trying to be super-futureproof to not have the package suddenly stop working in case React removed some core functionality such as `createClass` with 16.0.0 (or later), but since we aren't directly using it anyway (but as a mere peerDev), I guess we should be fine...